tools/bin/mtest

```diff
@@ -502,19 +507,23 @@ def get_crypto_config_file():
 
 def set_config_option(action, option, crypto_config_path, silent=True):
     force = [ "--force" ] if action == "set" else []
+    value = []
+    if action == "set":
+        if "=" in option:
+            option, v = option.split("=")
+            value = [v]
+        elif option.startswith("PSA_WANT"):
+            value = ['1']
     if option.startswith("PSA_WANT"):
         backup_config_options(crypto_config_path)
         if not silent:
-            log(f"scripts/config.py -f {crypto_config_path} {action} {option} {'1' if action == 'set' else ''}")
-        if action == "set":
-            output, success = subrun([f"scripts/config.py"] + force + ["--file", crypto_config_path, action, option, "1"], silent=True, ignore_failure=True)
-        else:
-            output, success = subrun([f"scripts/config.py"] + force + ["--file", crypto_config_path, action, option], silent=True, ignore_failure=True)
+            log(f"scripts/config.py -f {crypto_config_path} {action} {option} {''.join(value)}")
+        output, success = subrun([f"scripts/config.py"] + force + ["--file", crypto_config_path, action, option] + value, silent=True, ignore_failure=True)
```

Comment:
If force value are meant to be single element structures, we could use them as such str, int, and then convert in lists on the subrun

~~~~~~
output, success = subrun([f"scripts/config.py"] + [force] + ["--file", crypto_config_path, action, option] + [value], silent=True,
~~~~~~~